src/test/txvalidationcache_tests.cpp

```diff
@@ -156,7 +153,14 @@ BOOST_FIXTURE_TEST_CASE(checkinputs_test, TestChain100Setup)
     CScript p2sh_scriptPubKey = GetScriptForDestination(CScriptID(p2pkh_scriptPubKey));
     CScript p2wpkh_scriptPubKey = GetScriptForWitness(p2pkh_scriptPubKey);
 
-    const CTransactionRef p2pkh_coinbase = CreateAndProcessBlock({}, p2pkh_scriptPubKey).vtx[0];
+    bool processed;
+    const CTransactionRef p2pkh_coinbase = CreateAndProcessBlock({}, p2pkh_scriptPubKey, &processed).vtx[0];
+    BOOST_CHECK(processed);
+
+    // Make the reward with p2pkh_scriptPubKey mature
+    for (int i = 0; i < COINBASE_MATURITY + 1; ++i) {
```

Comment:
Instead of trying to use the staking output that now must be a witness, in order to test p2pkh sig features we use the reward from the first block, hence we need to make it mature before using it.